src/common/dto/meeting/MeetingResponse.dto.ts

```diff
@@ -0,0 +1,87 @@
+import { DAY } from 'common/constants';
+import { ApiModelProperty } from '@nestjs/swagger';
+
+/**
+ * Models the information a room included with the server response
+ */
+
+abstract class MeetingRoomResponse {
```

Comment:
On the first point: The building name should be included as part of the room name. I'll update the docs to make that clear

On the second point: I don't think we're showing the capacity in the context of the `Meeting`, so that's not needed here.

Reply:
Ok great!